tests/models/plants/data/all_config.toml

```diff
@@ -0,0 +1,51 @@
+# This file contains every tag required to build the config for the plants and soil
+# modules. Each value has been chosen to be different from the default value, so that
+# this file can be used to test that providing non-default values works.
+[core]
+modules = ["plants", "soil", "animals"]
+
+[core.grid]
+cell_nx = 10
+cell_ny = 10
+
+[core.timing]
+start_date = "2020-01-01"
+update_interval = "2 weeks"
+run_length = "50 years"
+
+[core.data_output_options]
+save_initial_state = true
+save_final_state = true
+out_initial_file_name = "model_at_start.nc"
+out_final_file_name = "model_at_end.nc"
+
+[plants]
+a_plant_integer = 12
```

Comment:
Now that there's a plant model it might be worth deleting this placeholder (from here and the schema) and switching the test that uses it to something you are actually using

Reply:
I agree but I'm expecting the schema to change more widely, so I might tidy that up in one pass when it has settled.

Reply:
Yeah fair, just thought I should mention it when I saw it